R/io.R

```diff
@@ -20,3 +20,58 @@ make_temporary_file <- function(content,extension) {
 read_temporary_file <- function(temp){
   return(readLines(temp))
 }
+
+#' This function creates a sample file
+#'
+#' @param save_filepath The path of the file that is being saved
+#' @param body The contents of the file
+#' @return file with specified path and body
+#' @export
+io_make_sample_file <- function(save_filepath,body) {
+
+  error <- system2('echo',
+                   args = c("\"print('hello world!')\"",
+                            '>',
+                            file_path),
+                   stdout = TRUE,
+                   stderr = FALSE)
```

Comment:
This function is adding as body the `print('hello world!') to the `file_path` variable, which is no longer defined in this function!